internal/QueryProcessor.py

```diff
@@ -1,4 +1,49 @@
 import requests
+from openai import OpenAI
+
+# pip install google-cloud-vision
+from google.cloud import vision
+# pip install --upgrade google-cloud-language
+from google.cloud import language_v2
```

Comment:
do something like `from google.cloud import language_v2 as language` to clean it up.

Reply:
See this. You don't *have* to, but I would highly suggest it for readability (and to ensure no one gives themself carpal tunnel).